### ISSUE TITLE
Update Terraform to support `app_service` and `db_service`

### ISSUE THREAD

**Issue by webvictim** [gravitational/teleport#6375]
### What
Teleport's HA Terraform example should support `app_service` and `db_service` as it currently does not.

### How
Assign some development time to work on it.

### Why
`app_service` has been in since Teleport 5 and `db_service` since Teleport 6, but the HA Terraform does not support it.

### Workaround
The only current workaround is for customers to manually update their infrastructure to support these and potentially build their own AMI.

**Comment by nitrocode** [comment]
The helm chart also doesn't support overriding these parameters. You have to fork the chart to add it to the agent.

**Comment by webvictim** [comment]
@nitrocode Unless I'm misunderstanding you, that's not really the case with the `teleport-kube-agent` chart. As per the docs:

https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-kube-agent#application-access
https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-kube-agent#database-access

You can't override the entire `app_service` or `db_service` section, but you can add as many individual apps/databases as you like via values and they will all be exposed by the agent.

**Comment by nitrocode** [comment]
Yes it's possible to add each database individually but to support adding all of them would be much easier to manage. In order to support all of them, we have to fork the chart to overwrite the teleport.yaml ConfigMap

**Comment by webvictim** [comment]
Also add support for `windows_desktop_service`, `discovery_service` and `okta_service`.
